src/App/components/file-dialog.model.ts

```diff
@@ -0,0 +1,89 @@
+// Copied from `electron.d.ts` since we can't import from 'electron' on the frontend.
```

Comment:
might be worth noting the version number you've copied from - up to you.

Reply:
Good idea, done.